Design-Docs/tools/scs-compatible.yaml

```diff
@@ -0,0 +1,35 @@
+name: SCS Compatible
+url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Design-Docs/tools/scs-compatible.yaml
+iaas:
+  versions:
```

Comment:
Do we really need another sub-key? I would prefer to directly start the array.

Reply:
We used to need the sub-key as there were (in an older version of the YAML standard) further keys on that level (e.g. `upcoming`, `lates`, ...). We could agree on ditching the extra key. What do you think @garloff?

Reply:
You mean like:
```yaml
iaas:
  - version: v2
    stabilized_at: yyyy-mm-dd
    [...]
  - version: v1
    stabilized_at: jjjj-mm-tt
```
Works for me.

Reply:
See branch feat/compliance-tool-drop-versions